packages/neos-ui-redux-store/src/CR/Nodes/index.js

```diff
@@ -192,7 +192,9 @@ export const reducer = handleActions({
             )
         ))
     ),
-    [FOCUS]: ({contextPath, fusionPath}) => $set('cr.nodes.focused', new Map({contextPath, fusionPath})),
+    [FOCUS]: ({contextPath, fusionPath}) => $all(
+        $set('cr.nodes.focused', new Map({contextPath, fusionPath}))
+    ),
```

Comment:
why wrap the $set in $all, if it is just 1 statement? I think the old code was more concise; and they are functionally equivalent, aren't they?